tests/integration/targets/aci_port_channel_member_policy/tasks/main.yml

```diff
@@ -0,0 +1,223 @@
+        - cm_update_port_channel_member_policy.current.0.lacpIfPol.attributes.descr == ''
+        - cm_update_port_channel_member_policy.current.0.lacpIfPol.attributes.prio == '32768'
+        - cm_update_port_channel_member_policy.current.0.lacpIfPol.attributes.txRate == 'normal'
+        - nm_update_port_channel_member_policy is changed
+        - cm_update_port_channel_member_policy.previous.0.lacpIfPol.attributes.name == 'ansible_port_channel_member_policy_1'
+        - cm_update_port_channel_member_policy.previous.0.lacpIfPol.attributes.descr == ''
+        - cm_update_port_channel_member_policy.previous.0.lacpIfPol.attributes.prio == '32768'
+        - cm_update_port_channel_member_policy.previous.0.lacpIfPol.attributes.txRate == 'normal' 
+        - nm_update_port_channel_member_policy.current.0.lacpIfPol.attributes.name == 'ansible_port_channel_member_policy_1'
+        - nm_update_port_channel_member_policy.current.0.lacpIfPol.attributes.descr == 'description_changed'
+        - nm_update_port_channel_member_policy.current.0.lacpIfPol.attributes.prio == '32700'
+        - nm_update_port_channel_member_policy.current.0.lacpIfPol.attributes.txRate == 'fast'
+        - nm_update_port_channel_member_policy_again is not changed
+        - nm_update_port_channel_member_policy_again.previous == nm_update_port_channel_member_policy_again.current
+        - nm_update_port_channel_member_policy_again.current.0.lacpIfPol.attributes.name == 'ansible_port_channel_member_policy_1'
+        - nm_update_port_channel_member_policy_again.current.0.lacpIfPol.attributes.descr == 'description_changed'
+        - nm_update_port_channel_member_policy_again.current.0.lacpIfPol.attributes.prio == '32700'
+        - nm_update_port_channel_member_policy_again.current.0.lacpIfPol.attributes.txRate == 'fast'
+
+  # QUERY
+
+  - name: Create another Port Channel Member policy
+    cisco.aci.aci_port_channel_member_policy:
+      <<: *create_port_channel_member_policy
+      name: ansible_port_channel_member_policy_2
+  
+  - name: Query Port Channel Member policy
+    cisco.aci.aci_port_channel_member_policy:
+      <<: *create_port_channel_member_policy
+      state: query
+    register: query_one
+
+  - name: Query all Port Channel Member policies
+    cisco.aci.aci_port_channel_member_policy:
+      <<: *aci_info
+      state: query
+    register: query_all
+
+  - name: Assert Port Channel Member policy query
+    ansible.builtin.assert:
+      that:
+        - query_one is not changed
+        - query_one.current | length == 1
+        - query_one.current.0.lacpIfPol.attributes.name == 'ansible_port_channel_member_policy_1'
+        - query_one.current.0.lacpIfPol.attributes.descr == 'description_changed'
+        - query_one.current.0.lacpIfPol.attributes.prio == '32700'
+        - query_one.current.0.lacpIfPol.attributes.txRate == 'fast'
+        - query_all is not changed
+        - query_all.current | length >= 2
```

Comment:
Add assertion statements to check the object name or dn, check the below statement for the reference.

```
          - query_all_ip_tags_1.current | length >= 2
          - "'uni/tn-ansible_test/eptags/epiptag-[2.2.2.2]-ansible_test_vrf' in query_all_ip_tags_1.current | map(attribute='fvEpIpTag.attributes.dn') | list"
          - "'uni/tn-ansible_test/eptags/epiptag-[1.1.1.1]-ansible_test_vrf' in query_all_ip_tags_1.current | map(attribute='fvEpIpTag.attributes.dn') | list"
```